### DISCUSSION TITLE
Começar estudos Inglês

### DISCUSSION THREAD

**Discussion by lipiw** [frontendbr/forum#1704] [Discussão]
Ola, me chamo Felipe e pretendo começar a estudar inglês da melhor forma, só não sei qual hahaha. Sei que bastante gente faz curso em escolas mas acho muito caro e sei que tem formas melhores na internet.

Gostaria de saber como estudam inglês, por algum aplicativo, site ou alguma escola de inglês?

**Comment by francysreymer** [comment]
Eu fiz exatamente o que esse cara fala no vídeo abaixo. Atualmente converso
com um indonésio e um egípcia todos os dias para melhorar a proficiência.
https://m.youtube.com/watch?v=F_RicZm1Ssk&t=380s

**Comment by alexguners** [comment]
@francysreymer usa qual app? to pensando em fazer algumas horas com o italk

**Comment by francysreymer** [comment]
Uso o site conversation exchange que não tem app. E o app Speaky que tem
site e um app incrível.

**Comment by felipe-heredia** [comment]
Atualmente, também estou aprendendo inglês.

O que faço é o seguinte:

1. Faço no mínimo 5 minutos de [Duolingo](https://pt.duolingo.com/) por dia.
2. Leio alguns conteúdos em inglês, como artigos, notícias etc.

> Explicando o 1: >> O Duolingo é bem básico, porém, fazendo diversos dias, você consegue conhecer o básico dos textos e quando for ler, entender pelo menos uns 30% da frase. Portanto, apenas o Duolingo não ajudará.

> Explicando o 2: >> Leio artigos, geralmente enviados pelo pessoal no grupo de discord da Rocketseat, a galera sempre envia artigos de React, ReactNative etc. Também, leio alguns artigos enviados todas as segundas, pela própia rocketseat, eles enviam e-mails com artigos para ler durante a semana.

As vezes, pego alguns textos que não entendo nada, porém, sempre ali com o Google tradutor do lado para entender.

### O importante é não desistir, se não conseguir ler hoje, salva o texto e leia outros todos os dias, depois de 2 meses volte e verá que entenderá um pouco mais daquele antigo.

Pratique, assim, entenderá mais.

Se alguém tentar de vender cursos "Inglês fluente em 7 dias", "Falando igual um nativo em 6 meses", algo desse tipo, pule fora, _não existe nada fácil_.
Nem mesmo um Nordestino consegue falar Português nativo, assim como na Bahia existem gírias, em São Paulo também existem. _"Parça"_, "_Suave_", "_Tranks_".

No máximo, com bastante estudo, conseguiremos entender 80% do texto, pois existem palavras que não existem traduções ou simplesmente gírias mesmo que os escritores usam.

**Comment by vsalbuq** [comment]
Fala aí, @lipiw ! Tudo certo?

Cara, vou colocar aqui o que fiz (e todo mundo precisa fazer, independente do método que escolha). Vai ser um textão. Peço desculpas de antemão, mas se eu escrevesse menos, seria incompleta a informação

Deve-se ter em mente que o idioma serve para um propósito: Comunicação. A comunicação, por sua vez é composta de duas partes que se subdividem em mais duas:

### Recepção
É como você absorve informação. As duas atividades que compõem esta parte são:
- Ler
- Ouvir

### Emissão
É como você transmite informação a outros. As duas atividades que compõem esta parte são:
- Escrever
- Falar

Você vai precisar trabalhar essas quatro partes para ter fluência em um idioma. Se você aprender como treinar cada uma delas e souber espaçar bem o treino ao longo da semana, vai conseguir proficiência em um tempo relativamente curto. Isso é o que todo método vai tentar fazer.

Agora vou para a implementação **pessoalmente minha**, que tem funcionado.

Se olharmos para a forma como uma criança aprende, veremos que ela começa pela recepção e aos poucos vai se arriscando na emissão. 

Quando ela comete um erro na recepção, o adulto a corrige. "Pega a caneta para mim, filho". Ele vai até o lápis e você diz "Não, do lado desse aí", e vai tentando até pegar a coisa certa. Quando ela comete um erro na emissão, o adulto também a corrige. A diferença em adultos aprendendo um idioma, você consegue se corrigir de outras formas.

O que quero dizer é: Na minha experiência pessoal aprendendo idiomas, identifiquei que, em geral, o melhor para mim é seguir um processo semelhante ao de uma criança aprendendo a se comunicar: 

1. Começo pela atividade de recepção. Porém, como já tenho conhecimento de outros idiomas, não preciso começar pelo "ouvir", que depende de agentes externos. A que menos depende de auxílios externos é a leitura, que uso inicialmente com o único propósito de formar vocabulário. É comum achar listas de palavras mais usadas por aí ("as 3 mil palavras mais usadas no inglês", e coisas do tipo). O objetivo do primeiro mês é apenas memorizar essas palavras, mas pode-se adicionar o "ouvir" com a tecnologia de text to speech. É comum que com menos de 5 mil palavras (o que não é muito) você consiga abranger mais de 70% do que é dito no cotidiano.

> **Bro-tip**: No firefox, abra um artigo em inglês no modo de leitura. No canto lateral esquerdo há a opção de leitura em voz alta. Basicamente, ele vai ler o artigo pra você, sinalizando em qual parágrafo ele está (o que pode te ajudar a começar a entender como as palavras soam). Text to speech em outros idiomas é bem ruim ainda, mas em inglês você pode usar sem medo.

2. Começo a ler algo grande no idioma. Geralmente é algum livro no Kindle. Ele tem uma função excelente chamada Word Wise, que coloca entre as linhas do texto, abaixo da palavra, uma definição mais simples (porém ainda em inglês) de palavras mais complexas. Isso ajuda MUITO. Até lendo a definição de uma palavra você continua praticando. Em casos mais extremos, ao selecionar a palavra ele mostra a definição no dicionário. Em última instância você pode até usar o tradutor embutido que há nele.

> **Bro-tip**: Se você escolher um livro que já conheça e goste, além de se manter motivado em lê-lo diariamente, vai ter um feedback muito maior de entendimento (pois o fato de já conhecer o conteúdo vai te ajudar a não cometer alguns erros comuns).

3. Quando estou começando a me sentir confortável na leitura, começo a assistir séries com legenda no idioma que estou aprendendo. Com isso eu aprendo melhor a sonoridade de cada palavra e tenho o apoio visual da boca se mexendo (frequentemente a melhor forma de pronunciar é simplesmente imitar o que fazem com a boca em vez de ficar tentando adivinhar como fazer o som).

> **Bro-tip**: Não se engane: É mais fácil assistir coisas com linguagem formal no início. Linguagem informal geralmente contém expressões idiomáticas, ditados e coisas de duplo sentido que dificultam bastante o entendimento do novato. Se você diz para alguém aprendendo Português "de grão em grão a galinha enche o papo" ele vai ficar perdido. Ele pode entender o sentido literal da frase, mas ainda assim o significado real será perdido. Quando eu estava aprendendo Inglês, usei a série House e foi bem tranquilo. Como a linguagem era formal, quase sempre quando eu pesquisava uma frase, ou traduzia palavra por palavra, o significado literal era o significado do que foi dito no contexto.

4. Nesse ponto eu já consigo ler e ouvir sem dificuldades, e o contato com o idioma já me permite imaginar com certo grau de certeza como dizer algo (de tanto ouvir outros dizendo a mesma coisa). Sabe quando você escreve algo em Português e pensa _"isso não parece certo"_, mesmo sem saber o que é, ou qual regra exatamente está sendo quebrada? É nesse ponto que eu começo a escrever coisas no idioma que estou aprendendo.

> **Bro-tip**: Escreva as coisas que estiver aprendendo no idioma. Nós temos um terreno muito fértil para isso, pois na nossa profissão estudaremos coisas novas para o resto da vida. Você pode fazer uma conta no Medium ou Dev.to e começar a escrever tutoriais simples. Isso pode até dar uma boa visibilidade e te alavancar profissionalmente, além de te estimular a compreender algo em um nível mais profundo, para conseguir explicar para outro.

5. Depois de ganhar alguma confiança escrevendo, começo a conversar (por texto) com nativos do idioma e peço para que me corrijam caso eu cometa alguma falha.

> **Bro-tip**: Aqui é o ponto em que você começa a depender muito de outras pessoas (todos os passos anteriores você pode fazer sozinho e se avaliar com certo grau de qualidade). Tendo em vista isso, vai ser bem difícil encontrar pessoas que façam isso de graça (a não ser que você seja um cara carismático). Se não quiser perder muito tempo aqui, pague por aulas de conversação no Cambly, Verbling ou iTalki. Vai ser muito mais produtivo, com toda certeza. Os passos anteriores não necessariamente serão melhores com algo pago, mas este é certo que será. Se mesmo assim quiser ir na faixa gratuita, existe o Tandem (um app que conecta pessoas que querem um aprender o idioma nativo do outro). Se você for simpático e respeitador, não terá problemas. As mulheres no app são meio arredias (com razão, porque tem muito cara escroto por aí), mas se você for um cara legal vai conseguir conversar com elas de boas também. **Tome cuidado com o que diz, pois as culturas são muito diferentes e algo pode ser tabu para a pessoa. Vá com calma.**

6. Juntamente com o passo anterior, começo a treinar a fala lendo artigos em voz alta no idioma que estou aprendendo, a fim de acostumar com a movimentação muscular da boca no idioma. No chat, você também pode pronunciar o que escrever e o que te falarem para treinar. É como andar de bicicleta. Você treina os movimentos até ficar tão acostumado que nem pensa mais ao executá-los.

> **Bro-tip**: O Rachel's English é um bom recurso para tirar a dúvida de como pronunciar as palavras. [Aqui](https://rachelsenglish.com/sound-chart/) você encontra uma representação escrita de como pronunciar alguns encontros vocálicos. [Aqui](https://rachelsenglish.com/video-categories/) tem um catálogo de vídeos agrupados por fonemas, mostrando como pronunciar algumas coisas.

7. Se fizer tudo isso, o último passo será apenas conversar com o máximo de pessoas nativas que conseguir. Se continuar seguindo o esquema de pagar por conversação, sugiro que troque de pessoa de tempos em tempos, a fim de absorver o máximo de sotaques, gírias e regiões diferentes. Se treinar apenas com norte-americano vai passar mal quando viajar para a Irlanda, por exemplo.

A última coisa que tenho para dizer é que é mais importante o contato constante com o idioma do que fazer coisas mirabolantes uma vez por semana. **Treinar 15 min. por dia é mais eficaz do que estudar apenas aos sábados por 3h.**

É isso...

Links:
- [Artigo excelente de um poliglota que entra em alguns pormenores que não entrei aqui](https://tim.blog/2014/03/21/how-to-learn-a-foreign-language-2/)
- [Auxílio para uma tonelada de idiomas](https://www.omniglot.com/writing/languages.htm)

**Comment by bbertolani** [comment]
Achei interessante os links que você enviou, adicionaria esses tb.

3000 words - bom para montar flashcards
https://www.ef.com/wwen/english-resources/english-vocabulary/top-3000-words/

Flashcard - aqui vai do gosto de cada um!
Anki <https://apps.ankiweb.net/>, NeuraCache <https://neuracache.com/>,
https://www.studyblue.com/

Youglish - motivo para ouvir diferentes pronuncias de videos do youtube,
basta pesquisar a palavra
youglish.com/  <http://youglish.com/>

Plugin do Chrome para Netflix, tradução, lista de palavras, muito bom mas é
pago
https://chrome.google.com/webstore/detail/language-learning-with-ne/hoombieeljmmljlkjmnheibnpciblicm

Grammarly, na versão free já é bom
 https://app.grammarly.com/ <https://app.grammarly.com/>

App para conversar, as vezes você acha alguem bem firmeza que auxilia
bastante
https://www.tandem.net/pt-br

Podcast/Youtube
James ESL - https://www.youtube.com/user/JamesESL - varias dicas de ingles
All Ears English Podcast -
https://open.spotify.com/show/6qXldSz1Ulq1Nvj2JK5kSR?si=pi2JCMuYTKilzuyAkvcaiQ

**Comment by LuanDc** [comment]
Curso Online eu to fazendo o da English Live, tem muito exercício prático, aulas particulares e aulas em grupos, ótimas pra treinar o aprendizado. Mas aprender mesmo é prática, dia a dia, então tenta ler artigos, livros, assistir filmes, não há maneira mais natural.

**Comment by conormonaghan** [comment]
Olá, Eu acho uma coisa muito importante é praticar como falar. Eu recommendo Cambly e Italki e pode ler mais aqui: https://oneminuteenglish.org/pt/cambly-ou-italki-qual-site-e-melhor-para-aprender-ingles/

**Comment by ghost** [comment]
**O que eu venho fazendo e percebi uma melhor muito grande na leitura...**
Faço duolingo apenas para copiar e colocar as frases no aplicativo AnkiWeb. 60 por dia (me custa 30 minutos por dia).
Reviso no anki.

- 100 cards já antigos

- 60 Cards Novos

*** Eu não faço mais duolingo como forma de aprendizado, motivo pelo qual ele demora a colocar os cards antigos para ser re-visto***

Para praticar a leitura eu leio perfil de Linkedin de desenvolvedores Brasileiros morando fora como forma de ver como foi a caminhada do mesmo até a saída do país.

Bom... Para o listening estou começando novamente o curso do Mairo Vegara que na verdade não é um curso e sim um metodo.
Resumindo o curso... Texto com audio curtos com inglês que de para entender, aconselho entrar no site dele tem textos muito bom lá de graça.) Depois de ouvir essa merda durante a semana, você pode lavar a louça pronunciando todo o texto de boa. E o engraçado que ao ver algum filme ou seriado aquela frase aparece do nada.

Comecei a ver também a serie Friends, não que eu goste. Mas é fácil acompanhar... O macete é pegar o texto do ep e estudar ele e depois vai assistindo em inglês com Lyrics com o tempo você percebe que até o Lyrics do netflix está errado. Então aconselho a procurar no google.

Ps: Se eu entrei no seu perfil do Linkedin, saiba que você está contribuindo com minha leitura de inglês. Agradeço desde já.
